client/app/src/Service/DocumentSyncService.php

```diff
@@ -151,7 +131,7 @@ public function syncSupportingDocument(QueuedDocumentData $documentData): ?Docum
 
     private function buildUpload(QueuedDocumentData $documentData)
     {
-        $fileName = $this->fileNameFixer->removeWhiteSpaceBeforeFileExtension($documentData->getFileName());
+        $fileName = FileNameFixer::removeWhiteSpaceBeforeFileExtension($documentData->getFileName());
```

Comment:
Should we also be removing unusual characters here? I would probably just fold all the normalisations into a single function. I don't really see why we need to have calls to different methods to normalise the filename, when we could do it in one pass.

Reply:
Mentioned the difference in the other comment around `[[[:blank:]]]`